### ISSUE TITLE
tab.open() hangs indefinitely

### ISSUE THREAD

**Issue by Leftium** [phantombuster/nickjs#7]
My NickJS script was working fine a few weeks ago. I don't think I changed anything, but it stopped working. I tried updating from NickJS 2.0 to 2.3 and Chrome dev to the latest version with no improvement.

Apparently it is hanging at the call to `tab.open(url)`. I also tried running the sample code at https://nickjs.org/, and it hangs at the same method. How do I figure out why NickJS is hanging there? (The PhantomBuster demo still works, so I think it must be something with my environment)

**Environment:**
- Windows 10 64-bit
- Node: v8.4.0
- NickJS 2.0, 2.3
- Chrome: 63.0.3236.0 (developer stream)
- *Executing from Git bash 2.12.0.windows.1*

**Simple code to reproduce (CoffeeScript):**

*Note the callback is never called.*

``` coffeescript
Nick = require 'nickjs'
nick = new Nick()

callback = (err, httpCode, httpStatus, url) =>
    console.log 'Callback:'
    console.log err

( () =>
    tab = await nick.newTab()
    console.log 'tab.open()'
    tab.open 'https://mixergy.com/interviews/', callback
)()
```

**Simple code to reproduce (JS):**

*Note the callback is never called.*

``` js
var Nick, callback, nick;

Nick = require('nickjs');

nick = new Nick();

callback = (err, httpCode, httpStatus, url) => {
  console.log('Callback:');
  return console.log(err);
};

(async() => {
  var tab;
  tab = (await nick.newTab());
  console.log('tab.open()');
  return tab.open('https://mixergy.com/interviews/', callback);
})();
```

**Comment by paps** [comment]
Hi @Leftium thanks for the report.

Can you try with Chrome 62? At Phantombuster we've stayed on the beta channel to guarantee better stability. That might be the reason.

**Comment by Leftium** [comment]
Yes! Things work again with Chrome Beta (version 62.0.3202.52).

Note the documentation at https://github.com/phantombuster/nickjs says Chrome 63+ is required. I think I installed the dev channel to ensure I met the minimum requirements. I'm not sure what dev version I originally installed, nor if Chrome dev automatically updated itself since the last successful run a few weeks ago.

So I'm still not sure why my environment worked a few weeks ago, then suddenly started failing without any changes I can think of. I'm just glad my NickJS script works again~ Thanks a lot! 😄

**Comment by paps** [comment]
Sorry about that, I will update the readme right away. Then when I have time I'll make sure it works both in Chrome Beta and Unstable.

Thanks again for the report 👍

**Comment by Leftium** [comment]
NickJS is hanging again. Chrome Beta must have updated itself to version 63.0.3236.0, which was the dev stream version that had this exact same problem.

And I thus I think I figured out how the original hanging started: The dev stream version worked until it secretly updated itself to version 63.0.3236.0...

I hope NickJS is fixed before 63.0.3236.0 becomes the current stable channel!

**Comment by paps** [comment]
I'm working on it. In the meantime you can set the CHROME_PATH env var to be the path of your standard (non-beta) Chrome 62.

**Comment by paps** [comment]
This is fixed in 0.2.4 :)
